scripts/production/run_datachecks.pl

```diff
@@ -0,0 +1,158 @@
+
+=head1 SYNOPSIS
+
+  perl run_datachecks.pl $COMPARA_REG compara_curr -group compara
+
+  perl run_datachecks.pl \
+    --reg_conf registry_configuration_file --reg_alias compara_curr \
+    -name CompareMSANames
+
+=head1 ARGUMENTS
+
+The script reads these arguments and passes the other ones straight to run_datachecks.pl
+
+=head2 DATABASE SETUP
+
+=over
+
+=item B<[--url mysql://user[:passwd]@host[:port]/dbname]>
+
+URL of the database to test.
+
+=item B<[--reg_conf registry_configuration_file]>
+
+The Bio::EnsEMBL::Registry configuration file. If none given and no URL is
+given, the one set in ENSEMBL_REGISTRY will be used if defined, if not
+~/.ensembl_init will be used.
+
+=item B<[--reg_type reg_type]>
+
+The "type" or "group" under which the database is to be found in the Registry.
+
+=item B<[--reg_alias|--reg_name name]>
+
+The name or "species" under which the database is to be found in the Registry.
+
+=item B<[--prev_url mysql://user[:passwd]@host[:port]/dbname]>
+
+URL of the previous database.
+
+=item B<[--prev_alias|--prev_reg_name name]>
+
+The name or "species" under which the previous database is to be found in the Registry.
+Defaults to "compara_prev".
+
+=back
+
+=head2 DATACHECK SETUP
+
+=over
+
+=item B<[--dc-runner /path/to/run_datachecks.pl]>
+
+The path to the run_datachecks.pl of ensembl-datacheck. If not given a
+default will be formed using the ENSEMBL_CVS_ROOT_DIR environment variable.
+
+=back
+
+=cut
+
+use strict;
+use warnings;
+
+use Getopt::Long qw(:config pass_through);
+
+use Bio::EnsEMBL::Registry;
+
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::Compara::Utils::Registry;
+
+my ($url, $reg_conf, $reg_type, $reg_alias);
+my ($prev_url, $prev_alias);
+my ($dc_runner);
+
+# Arguments parsing
+GetOptions(
+    'url=s'                         => \$url,
+    'reg_conf|regfile|reg_file=s'   => \$reg_conf,
+    'reg_type=s'                    => \$reg_type,
+    'reg_alias|regname|reg_name=s'  => \$reg_alias,
+    'prev_url=s'                    => \$prev_url,
+    'prev_alias|prev_reg_name=s'    => \$prev_alias,
+    'dc-runner=s'                   => \$dc_runner,
+);
+
+unless (($url and $prev_url) or ($reg_conf and $reg_alias)) {
+    print "\nThe script requires either URLs (--url, --prev_url) or a registry configuration (--reg_conf, --reg_alias, --prev_alias).\n\n";
+    exit 1;
+}
+
+unless ($dc_runner) {
+    die "Need to give the --dc-runner option or set the ENSEMBL_CVS_ROOT_DIR environment variable to use the default" unless $ENV{ENSEMBL_CVS_ROOT_DIR};
+    $dc_runner = $ENV{ENSEMBL_CVS_ROOT_DIR} . '/ensembl-datacheck/scripts/run_datachecks.pl';
+}
+die "'$dc_runner' is not a valid executable" unless -x $dc_runner;
+
+if ($reg_conf) {
+    Bio::EnsEMBL::Registry->load_all($reg_conf);
+}
+my $dba = $reg_alias
+    ? Bio::EnsEMBL::Registry->get_DBAdaptor( $reg_alias, $reg_type || 'compara' )
+    : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->new( -URL => $url );
```

Comment:
I must confess I have found it interesting how in the previous two scripts you were testing for `$url` and in this one you have gone for `$reg_alias` :nerd_face: Thinking about it more thoroughly, do we want `$url` be the main parameter (in case both are passed)? Should we set a consensus in our scripts?

Reply:
Functionality is not affected. I don't think we need a consensus.

Reply:
I must confess I'm slightly concerned that we will have two scripts that, if you pass both `$reg_alias` and `$url` (by mistake, I assume), they will behave differently. But then again, only one should be passed, so we can ignore my previous comment.

Reply:
Right, that's an understandable concern. I think it should be addressed by making the script die if both are set